src/reducers/comments.spec.tsx

```diff
@@ -637,4 +641,287 @@ describe(__filename, () => {
+      const store = configureStore();
+      const versionId = 1;
+      store.dispatch(actions.beginLoadVersionComments({ versionId }));
+
+      const { dispatch, thunk } = thunkTester({
+        createThunk: () => _fetchAndLoadComments({ versionId }),
+        store,
+      });
+
+      await thunk();
+
+      expect(dispatch).not.toHaveBeenCalled();
+    });
+
+    it('handles API errors', async () => {
+      const error = new Error('API Error');
+      const _getComments = jest.fn().mockResolvedValue({ error });
+      const versionId = 1;
+
+      const { dispatch, thunk } = thunkTester({
+        createThunk: () => _fetchAndLoadComments({ _getComments, versionId }),
+      });
+
+      await thunk();
+
+      expect(dispatch).toHaveBeenCalledWith(
+        actions.abortLoadVersionComments({ versionId }),
+      );
+      expect(dispatch).toHaveBeenCalledWith(errorsActions.addError({ error }));
+    });
+
+    it('fetches version comments', async () => {
+      const _getComments = jest
+        .fn()
+        .mockResolvedValue(createFakeCommentsResponse());
+      const store = configureStore();
+      const addonId = 1;
+      const versionId = 1;
+
+      const { thunk } = thunkTester({
+        createThunk: () =>
+          _fetchAndLoadComments({ _getComments, addonId, versionId }),
+        store,
+      });
+
+      await thunk();
+
+      expect(_getComments).toHaveBeenCalledWith({
+        addonId,
+        apiState: store.getState().api,
+        versionId,
+      });
+    });
+
+    it('loads comments for a version + file + line', async () => {
+      const versionId = 1;
+
+      const commentId1 = 1;
+      const commentId2 = 2;
+      const commentId3 = 2;
+
+      const fileName = 'manifest.json';
+      const line = 123;
+
+      const extComments = [commentId1, commentId2, commentId3].map((id) => {
+        return createFakeExternalComment({
+          id,
+          version: { ...fakeVersion, id: versionId },
+          filename: fileName,
+          lineno: line,
+        });
+      });
+
+      const _getComments = jest
+        .fn()
+        .mockResolvedValue(createFakeCommentsResponse(extComments));
+
+      const { dispatch, thunk } = thunkTester({
+        createThunk: () => _fetchAndLoadComments({ _getComments, versionId }),
+      });
+
+      await thunk();
+
+      expect(dispatch).toHaveBeenCalledWith(
+        actions.setComments({
+          versionId,
+          fileName,
+          line,
+          comments: extComments,
+        }),
+      );
+    });
+
+    it('loads comments for a version, file, and a line', async () => {
+      const versionId = 1;
+      const extVersion = { ...fakeVersion, id: versionId };
+
+      const commentId1 = 1;
+      const commentId2 = 2;
+      const commentId3 = 2;
+      const commentId4 = 4;
+      const commentId5 = 5;
+      const commentId6 = 6;
+
+      const fileName = 'manifest.json';
+      const line = 321;
+
+      const extComment = ({
```

Comment:
I suppose `ext` means `external` but that's not really obvious. Could you rename this function by chance? `createExternalComment` would work well, I guess.

Reply:
I think a longer name would actually reduce the readability of this since the reader has to parse more words. Since the function is only used for a few lines, I wanted to keep it short.

Reply:
I'll remove the `ext` part